db/schema.rb

```diff
@@ -29,8 +29,11 @@
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
     t.string   "unconfirmed_email"
-    t.datetime "created_at",                          null: false
-    t.datetime "updated_at",                          null: false
+    t.datetime "created_at",                             null: false
+    t.datetime "updated_at",                             null: false
+    t.boolean  "admin",                  default: false
```

Comment:
I would prefer to see a role column that contained an integer that maps to a role. I.e,

```
0 = standard
1 = premium
2 = standard
```

Rails has a built in way of handling these types of situations: [the enum](http://api.rubyonrails.org/classes/ActiveRecord/Enum.html).